symfony/messenger/4.3/config/packages/messenger.yaml

```diff
@@ -0,0 +1,13 @@
+framework:
+    messenger:
+        # Uncomment the following line and choose a transport to collect messages that fail so you can retry them later
+        # failure_transport: ...
```

Comment:
What about giving an example with Doctrine here?

Reply:
There is an example in the `manifest.json`, but I can add a "failed" transport that will use Doctrine ?

Reply:
That's what I meant, having a failed transport using Doctrine (which I suppose will be the most common configuration).

Reply:
Let’s try it. So we would add a transport, maybe called “failed” and show it using a doctrine dsn - maybe also with ?queue=failed (so that it doesn’t collide with a real transport of the user also uses doctrine for that). Double check that config is correct. Then, this line (still commented), would be failure_transport: failed

Reply:
@weaverryan, I updated my PR just at the same time as your comment. What do you think of my suggestion with a new env var `MESSENGER__FAILED_TRANSPORT_DSN ` which contains the configuration ?